integrations/pooltogether-v4/README.md

```diff
@@ -0,0 +1,27 @@
+# PoolTogether TurboETH Integration
+
+
+## Features
+- Deposit on PoolTogether V4
+- Withdraw from PoolTogether V4
+---
+
+## File Structure
+```
+integrations/pooltogethet-v4
+├─ abis/
+│  ├─ yield-source-prize-pool-abi.ts
+│  ├─ yield-source-prize-pool-bytecode.json
+├─ components/
+│  ├─ form-yield-source-prize-pool-deposit.tsx
+│  ├─ form-yield-source-prize-pool-withdraw.tsx
+├─ hooks/
+│  ├─ use-load-contract-from-chain-id.tsx
+│  ├─ use-usdc-approval.tsx
+│  ├─ use-user-balance-deposit.tsx
+│  ├─ use-user-balance-withdraw.tsx
+├─ prize-pool-contract-list.ts
```

Comment:
Just a suggestion, but moving these 3 `*-contract-list.ts` to a `utils` folder would make the file structure cleaner